packages/checkout/widgets-lib/src/widgets/add-funds/hooks/useRoutes.ts

```diff
@@ -52,6 +55,15 @@ export const useRoutes = () => {
     return fromAmountWithBuffer.toString();
   };
 
+  const calculateFromAmountFromRoute = (
+    exchangeRate: string,
+    toAmount: string,
+  ) => {
+    const fromAmount = Number(toAmount) / Number(exchangeRate);
+    const fromAmountWithBuffer = fromAmount * 1.02;
```

Comment:
should we exract the 1.02 number as a const on top? seems like it's being used in both `calculateFromAmountFromRoute` and `calculateFromAmount` 👀